plugin.rb

```diff
@@ -27,7 +27,8 @@
   end
 
   add_to_class(:guardian, :can_create_jira_issue?) do
-    SiteSetting.discourse_jira_enabled && is_staff?
+    SiteSetting.discourse_jira_enabled &&
```

Comment:
This is what you were missing, I pushed a commit to fix it. Previously this was just `is_staff?` and this line is hit by this line in the controller:

https://github.com//hub.com/discourse/discourse-jira.git/blob/aad2ee200cacb904a690bb3134f2a0ab0e7f11ef/app/controllers/discourse_jira/issues_controller.rb#L254-L256

Now that this is checking the groups, we can also change the `add_to_serializer` for `can_create_jira_issue` to just check this guardian method instead

Reply:
d'oh!!! thank you!